src/main/java/org/jabref/gui/preferences/autocompletion/AutoCompletionTabViewModel.java

```diff
@@ -121,4 +124,24 @@ public BooleanProperty firstNameModeFullProperty() {
     public BooleanProperty firstNameModeBothProperty() {
         return firstNameModeBothProperty;
     }
+
+    public StringConverter<Field> getFieldStringConverter() {
+        return new StringConverter<>() {
+            @Override
+            public String toString(Field field) {
+                return field.getDisplayName();
+            }
+
+            @Override
+            public Field fromString(String string) {
+                return FieldFactory.parseField(string);
+            }
+        };
+    }
+
+    public List<Field> getSuggestions(String request) {
+        return FieldFactory.getStandardFields().stream()
```

Comment:
Better use getAllFieldsWithOutInternal
1. it contains more than bibtex and biblatex fields (the additional fields for other entry types, e.g. Software or IEE)
2. they are already sorted

Reply:
Done